src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/SentReferralSummaryForDashboardDTO.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ReferralForDashboard
+import java.time.OffsetDateTime
+import java.util.UUID
+
+data class SentReferralSummaryForDashboardDTO(
+  val id: UUID,
+  val sentAt: OffsetDateTime,
+  val sentBy: AuthUserDTO,
+  val referenceNumber: String,
+  val assignedTo: AuthUserDTO?,
+  val serviceUser: ServiceUserDTO?,
+  val serviceProvider: ServiceProviderDTO?,
+  val supplementaryRiskId: UUID,
+  val interventionTitle: String,
+  val concludedAt: OffsetDateTime?
+) {
+  companion object {
+    fun from(referral: ReferralForDashboard): SentReferralSummaryForDashboardDTO {
+      return SentReferralSummaryForDashboardDTO(
+        id = referral.id,
+        sentAt = referral.sentAt!!,
```

Comment:
it would be cool if this new entity projection has these fields as non-nullable. 

the only reason they are nullable in the existing entity is that it is shared by referrals that in both draft and sent states. all referrals here will be 'sent' - i think !

Reply:
So in the landing page, we won't show the Draft Referrals?